ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -280,6 +280,10 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
         NotificationCenter.default.addObserver(self, selector: #selector(handleContentSizeCategoryDidChange), name: UIContentSizeCategory.didChangeNotification, object: nil)
     }
 
+    @objc open func setup(style: Style, title: String, accessoryButtonTitle: String = "") {
```

Comment:
Could you add Xcode style documentations to these setup method overloads that are exposed to objc?

Reply:
also the simpliest variation we care about might be just style and title ? what is most used variation in office?

Reply:
It seems like all the usage is indeed only style and title. I agree that providing another overload with these 2 params could be very useful.